src/components/shared/Home/PaginationButtons.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+import {
+  Pagination,
+  PaginationContent,
+  PaginationItem,
+  PaginationLink,
+} from "@/components/ui/pagination";
+import React from "react";
+
+interface PaginationButtonsProps {
+  page: string | undefined;
+  totalPages: number;
+}
+
+const PaginationButtons: React.FC<PaginationButtonsProps> = ({
+  page,
+  totalPages,
+}) => {
+  const pageNum = parseInt(page || "1");
+  return (
+    <Pagination>
+      <PaginationContent>
+        {pageNum !== 1 && (
+          <PaginationItem>
+            <PaginationLink
+              className="bg-myBlack-700 hover:bg-myBlack-600"
+              href={`/?page=${pageNum - 1}`}
+            >
+              Prev
+            </PaginationLink>
+          </PaginationItem>
+        )}
+        <PaginationItem>
+          <p className="text-p3Med text-myWhite-300">
+            {page} /{totalPages}{" "}
+          </p>
+        </PaginationItem>
+        <PaginationItem>
+          {pageNum !== totalPages && (
+            <PaginationItem>
+              <PaginationLink
+                className="bg-myBlack-700 hover:bg-myBlack-600"
+                href={`/?page=${pageNum + 1}`}
+              >
+                Next
+              </PaginationLink>
+            </PaginationItem>
+          )}
+        </PaginationItem>
```

Comment:
![image](https://github.com/dalbanhi/git-note/assets/4108484/9cbd38bc-9e52-4ab7-9b71-1bff756d4e74)

We're getting an error here because of the nested "Pagination Item" components.

Let's remove one